src/ui/actions/session.ts

```diff
@@ -320,7 +320,6 @@ export function createSocket(
       // We don't want to show the non-dev version of the app for node replays.
       if (recordingTarget === "node") {
         dispatch(setViewMode("dev"));
-        dispatch(onLoadingFinished());
```

Comment:
How did we end up dispatching this _twice_ in the case where it's a Node recording?  Looks like.... Jason added that in June 2022.

Reply:
Haha, yeah I dunno, many hands make light work... but also sometimes the work is light and also a little confused!